src/main/resources/db/migration/local+dev/R__2_create_premises.sql

```diff
@@ -0,0 +1,12 @@
+-- ${flyway:timestamp}
+TRUNCATE TABLE premises CASCADE;
+
+insert into "premises" ("address_line1", "id", "local_authority_area_id", "name", "notes", "postcode", "probation_region_id", "service", "status", "total_beds") values ('Address 1', 'd33006b7-55d9-4a8e-b722-5e18093dbcdf', 'd75ce5b8-fc07-494b-8950-a46a63ac377e', 'Something House', NULL, 'LA9 1DS', 'a02b7727-63aa-46f2-80f1-e0b05b31903c', 'temporary-accommodation', 'active', 30);
```

Comment:
@danlivings-dxw how does TA use total_beds? Do we set it to 0 and not use it or do we update it after the creation of a bedspace? We should set these seed values to something realistic for TA.

Reply:
@tahb It just gets set to 0 when the premises is created - the value is calculated by counting the beds in rooms associated with a premises.